src/screens/auth/SignIn.tsx

```
@@ -55,6 +55,21 @@ const SignIn: React.FC = () => {
 					'refreshToken',
 					response.data.refresh_token
 				);
+				localStorage.setItem(
+					'walletToken',
+					response.data.walletToken
+				);
```

ESLint

[error] Replace `⏎↹↹↹↹↹'walletToken',⏎↹↹↹↹↹response.data.walletToken⏎↹↹↹↹` with `'walletToken',·response.data.walletToken`

(prettier/prettier)